Week 9 Documentation Updates (#68)
AY2223S1-CS2103T-W15-1/tp: docs/UserGuide.md

Example:
* `customise show a/` The application now shows addresses in the contact list.
* `customise show p/ t/` The application now shows phone numbers and tags in the contact list.

### Customise Order of Details: `customise order`

Customise order of information shown in all contacts shown.

Format: `customise order [p/] [e/] [a/] [t/]`

* Attributes not given will be ordered last in default order. (tag > phone > email > address)

Example:

> * `customise order a/ e/ p/` The application will show address first, followed by email, phone, then tags.

Reviewer: ```suggestion
* `customise order a/ e/ p/` The application will show address first, followed by email, phone number, then tags.
```

Author: Ok, I have added it.

> * `customise order a/` The application now shows address first the rest of the attributes will follow the default order, therefore address will be followed by tags, phone and then email.

Reviewer 2: ```suggestion
* `customise order a/` The application will show address first. The rest of the attributes will follow the default order. Therefore address will be followed by tags, phone and then email.
```

Author: Ok, I have changed it.

> * `customise order a/` The application will show address first. The rest of the attributes will follow the default order, therefore address will be followed by tags, phone and then email.

Reviewer: ```suggestion
* `customise order a/` The application will show address first. The rest of the attributes will follow the default order, therefore address will be followed by tags, phone number and then email.
```

Author: Ok, I have added it.

|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **Add**        | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague` |
| **Clear**      | `clear`                                                                                                                                                               |

> | **Customise**  | `customise hide [t/] [p/] [e/] [a/]` <br> `customise show [t/] [p/] [e/] [a/]` <br> `customise order [t/] [p/] [e/] [a/]` <br> e.g, `customise order a/ e/ p/`        |

Reviewer: Add examples for the other 2 subcommands?

Author: Okay, I shall add examples for the other two.
